### PR TITLE
Fixes #11257: Do not send inventory if a forwarder/uploader daemon exists

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11257

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/propagatePromises.st [hunk 1]
```diff
@@ -183,6 +183,7 @@ bundle agent sendInventoryToCmdb
     any::
       "incoming_inventory_files" slist => findfiles("${g.rudder_inventories}/incoming/*.ocs", "${g.rudder_inventories}/incoming/*.ocs.gz");
       "updates_inventory_files" slist => findfiles("${g.rudder_inventories}/accepted-nodes-updates/*.ocs", "${g.rudder_inventories}/accepted-nodes-updates/*.ocs.gz");
+      "disable_upload" string => fileexists("/opt/rudder/etc/no_inventory_upload");
```
Comment (ncharles) [inline techniques/system/distributePolicy/1.0/propagatePromises.st]: this should be in a classes: section, with the syntax
`"disable_upload" expression => fileexists("/opt/rudder/etc/no_inventory_upload");`

it is used in several bundle, so it should either be in a bundle common, or with a scope namespace

Comment (peckpeck) [reply techniques/system/distributePolicy/1.0/propagatePromises.st]: oops indeed for the class
but it is used in a single bundle

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**